electron-app/package.json

```diff
@@ -24,7 +24,7 @@
     "release:osx": "export $(cat .env | xargs) && electron-builder -m -p always",
     "clean": "rimraf release",
     "format": "prettier --write \"src/**/*.ts\" \"test/**/*.ts\"",
-    "start": "npm run prebuild && nest build && electron dist/main --develop",
+    "start": "run-s prebuild build start:electron",
```

Comment:
I don't see a run all dependency in this package fle.

Reply:
> I don't see a run all dependency in this package fle.

npm resolves it from ../

that also why i started all of that

Reply:
@mvdicarlo